common/c_cpp/src/c/mempool.c

```diff
@@ -76,6 +76,7 @@ memoryNode*
 memoryPool_addNode (memoryPool* pool, size_t nodeSize)
```

Comment:
Ideally yes, but if that's painful to retro-fit, I might be able to do some merge magic to spoof combining the two efforts instead when it comes to getting these into next.